docs/help_out.txt

```diff
@@ -43,34 +50,34 @@ GLOBAL OPTIONS:
+   --eigenda.v2.blob-certified-timeout value     Maximum amount of time to wait for blob certification against the on-chain EigenDACertVerifier. (default: 30s) [$EIGENDA_PROXY_EIGENDA_V2_CERTIFY_BLOB_TIMEOUT]
+   --eigenda.v2.blob-status-poll-interval value  Duration to query for blob status updates during dispersal. (default: 1s) [$EIGENDA_PROXY_EIGENDA_V2_BLOB_STATUS_POLL_INTERVAL]
+   --eigenda.v2.blob-version value               Blob params version used when dispersing. 
+                                                            This refers to a global version maintained by EigenDA governance 
+                                                            and is injected in the BlobHeader before dispersing.
+                                                            Currently only supports (0). (default: 0) [$EIGENDA_PROXY_EIGENDA_V2_BLOB_PARAMS_VERSION]
+   --eigenda.v2.cert-verifier-addr value         Address of the EigenDACertVerifier contract. Required for performing eth_calls to verify EigenDA certificates. [$EIGENDA_PROXY_EIGENDA_V2_CERT_VERIFIER_ADDR]
+   --eigenda.v2.contract-call-timeout value      Timeout used when performing smart contract call operation (i.e, eth_call). (default: 10s) [$EIGENDA_PROXY_EIGENDA_V2_CONTRACT_CALL_TIMEOUT]
+   --eigenda.v2.disable-point-evaluation         Disables IFFT transformation done during payload encoding. Using this mode results in blobs that can't be proven. (default: false) [$EIGENDA_PROXY_EIGENDA_V2_DISABLE_POINT_EVALUATION]
+   --eigenda.v2.disable-tls                      Disable TLS for gRPC communication with the EigenDA disperser and retrieval subnet. (default: false) [$EIGENDA_PROXY_EIGENDA_V2_GRPC_DISABLE_TLS]
+   --eigenda.v2.disperse-blob-timeout value      Maximum amount of time to wait for a blob to disperse against v2 protocol. (default: 2m0s) [$EIGENDA_PROXY_EIGENDA_V2_DISPERSE_BLOB_TIMEOUT]
+   --eigenda.v2.disperse-to-v2                   Enable blob dispersal and retrieval against EigenDA V2 protocol. (default: false) [$EIGENDA_PROXY_EIGENDA_V2_DISPERSE_TO_V2]
+   --eigenda.v2.disperser-rpc value              RPC endpoint of the EigenDA disperser. [$EIGENDA_PROXY_EIGENDA_V2_DISPERSER_RPC]
+   --eigenda.v2.eth-rpc value                    URL of the Ethereum RPC endpoint. [$EIGENDA_PROXY_EIGENDA_V2_ETH_RPC]
+   --eigenda.v2.max-blob-length value            Maximum blob length to be written or read from EigenDA. Determines the number of SRS points
+                                                           loaded into memory for KZG commitments. Example units: '30MiB', '4Kb', '30MB'. Maximum size
+                                                           slightly exceeds 1GB. (default: "16MiB") [$EIGENDA_PROXY_EIGENDA_V2_MAX_BLOB_LENGTH]
```

Comment:
Just flagging this, no need to do anything about it right now.
First of all this file is super helpful.. quite a nice addition!

That being said this part is quite hard to review. I think only 1 or 2 flags changed but the diff being presented is prob doing some stupid line-by-line diff instead of structural diff. Think if we output to a markdown we might be able to get a nicer diff presentation? Not sure.. cc @ethenotethan

Reply:
Yeah, I'm not sure why github is so bad at showing this diff... the tool I use locally handles this in a sane way, and it's easy enough to review.